### PR TITLE
Require ^3.3.8 for Symfony - when in 3.3.x

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | mentioned in #8565, closing #8570  |
| License         | MIT |

Based on @stefandoorn work, just to merge it faster.

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Application/Kernel.php [hunk 1]
```diff
@@ -97,17 +97,14 @@ public function registerBundles(): array
             new \Sylius\Bundle\FixturesBundle\SyliusFixturesBundle(),
             new \Sylius\Bundle\PayumBundle\SyliusPayumBundle(), // must be added after PayumBundle.
             new \Sylius\Bundle\ThemeBundle\SyliusThemeBundle(), // must be added after FrameworkBundle
+
+            new \Symfony\Bundle\WebServerBundle\WebServerBundle(), // allows to run build in web server. Not recommended for prod environment
```
Comment (loic425) [inline src/Sylius/Bundle/CoreBundle/Application/Kernel.php]: why moving this here ?

Comment (pjedrzejewski) [reply src/Sylius/Bundle/CoreBundle/Application/Kernel.php]: We decided that it does not bring much optimization but is useful when you have an issue appearing only on prod env. We still run apps in prod locally from time to time to discover a problem.

**Comment by pjedrzejewski** [comment]
Thank you Łukasz!

File: src/Sylius/Bundle/CoreBundle/Application/Kernel.php (hunk 1 above)
Comment (pamil) [reply src/Sylius/Bundle/CoreBundle/Application/Kernel.php]: To make it available in another environments like staging (and nobody's forced to use it on prod ofc, it's just an easy way to see if things are working correctly without setting up a webserver).

Comment (loic425) [reply src/Sylius/Bundle/CoreBundle/Application/Kernel.php]: ok I understand the reason. I was just surprised cause I've seen the previous pull-request moving it on dev/test environment :)

File: bin/symfony_requirements [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env php
 <?php
 
-require_once dirname(__FILE__).'/../var/SymfonyRequirements.php';
```
Comment (bendavies) [inline bin/symfony_requirements]: @lchrusciel what's going on here? this path is now wrong.

Comment (bendavies) [reply bin/symfony_requirements]: https://github.com/sensiolabs/SensioDistributionBundle/issues/324

Comment (lchrusciel) [reply bin/symfony_requirements]: My bad, on it

**Pull request merged**